MonoGame.Framework/Audio/OpenALSoundController.cs

```diff
@@ -237,15 +237,7 @@ private bool OpenSoundController()
 #elif IOS
                 AVAudioSession.SharedInstance().Init();
 
-                // This is the default audio session category on iOS.
-                //
-                //   Your audio is silenced by screen locking and by the Silent switch (called the Ring/Silent switch on iPhone).
-                //
-                //   By default, using this category implies that your app�s audio is nonmixable�activating your session will
-                //   interrupt any other audio sessions which are also nonmixable.
-                //
-                //
-                AVAudioSession.SharedInstance().SetCategory(AVAudioSessionCategory.SoloAmbient);
+                // NOTE: Do not override AVAudioSessionCategory set by the game developer per #6595.
```

Comment:
You don't need to add this comment.

Reply:
The comment is intended to reduce the chance that someone adds that code again, which seems to be happening over the history of the project.

Reply:
Changed it to a URL, I'll leave it in.